packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -243,11 +243,17 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 			}
 		});
 
+		const immediatelyRefreshLatestSummaryAck =
+			this.mc.config.getBoolean("Fluid.Summarizer.immediatelyRefreshLatestSummaryAck") ??
+			false;
 		this.generator = new SummaryGenerator(
 			this.pendingAckTimer,
 			this.heuristicData,
 			this.submitSummaryCallback,
-			() => {
+			async (options: IRefreshSummaryAckOptions) => {
```

Comment:
Can you separate this logic into a separate PR. This will make it easier to review and also help back out or pick this part in case of any requirements.

Reply:
Will do